namedtensor/torch_base.py

```diff
@@ -67,6 +67,12 @@ def topk(tensor, dim, k, largest=True, sorted=True):
             ntorch.tensor(arg_top_k, old_names),
         )
 
+    @staticmethod
+    def chunk(tensor, chunks, dim):
+        old_names = tensor._schema._names
+        chunks = tensor._tensor.chunk(chunks, dim=tensor._schema.get(dim))
+        return [ntorch.tensor(chunk, old_names) for chunk in chunks]
```

Comment:
Not sure I understand why "chunks" is an argument here? 

Also instead of using `ntorch.tensor()`  can you use tensor._new  (see others). It will default to the same args.